### ISSUE TITLE
Create action for data dumping

### ISSUE THREAD

**Issue by hikmat30ce** [jamband/yii2-schemadump#14]
Hi Great work and I have enjoyed it when migrating my websites from local to remote servers, but found that there is no option for copying data from database. if this option is added then it will be very good

**Comment by jamband** [comment]
I agree so, But I think that it is difficult to implement it. Please tell me if there is a good idea :)

**Comment by hikmat30ce** [comment]
I have seen few plugins that works and produce the insert starments from tables  but they have other issues. you can check https://github.com/c006/yii2-migration-utility

**Comment by hikmat30ce** [comment]
I can offer my contribution if you allow me

**Comment by hikmat30ce** [comment]
I can offer my contribution if ok with you

**Comment by jamband** [comment]
@hikmat30ce Thank you. Please pull request :)

**Comment by jamband** [comment]
sorry. It is a mistake :(
